### PR TITLE
Allows species and tongue to modify yell/ask/exclaim

### PR DESCRIPTION
**Pull request by BootlegBow**
## About The Pull Request

It's just #58981 but this time without me getting banned for a shitty meme PR

## Why It's Good For The Game

if the voicebox is robotic why does it perfectly mimic a human asking a question huh?

## Changelog
:cl:
code: yell/ask/exclaim mods based on species or tongue
qol: cyber voicebox now has all the same speechmods as a borg
/:cl:

### CODE REVIEW

File: code/modules/surgery/organs/tongue.dm [hunk 1]
```diff
@@ -40,6 +43,11 @@
 
 /obj/item/organ/tongue/Insert(mob/living/carbon/tongue_owner, special = 0)
 	..()
+	if(tongue_owner.dna && tongue_owner.dna.species)
+		tongue_owner.dna.species.say_mod = say_mod
+		tongue_owner.dna.species.ask_mod = ask_mod
+		tongue_owner.dna.species.exclaim_mod = exclaim_mod
+		tongue_owner.dna.species.yell_mod = yell_mod
```
Comment (Time-Green) [inline code/modules/surgery/organs/tongue.dm:50]: Wouldn’t it be easier to delegate the species say mods to whatever tongue they are created with, and then have the speech code directly use the tongues speech mod? That way you don’t have to keep overriding the species datum (which you is not meant to be overridden), the speech wont revert on species change and the code would be a lot more future proof

**Review by Time-Green — CHANGES_REQUESTED** [review]

**Comment by TheBonded** [comment]
> It's just #58981 but this time without me getting banned for a shitty meme PR

What was the original part of that PR that got you the 30 day ban? I remember seeing that PR, I never checked the code but I figured it'd be a good change

**Comment by BootlegBow** [comment]
> > It's just #58981 but this time without me getting banned for a shitty meme PR
> 
> What was the original part of that PR that got you the 30 day ban? I remember seeing that PR, I never checked the code but I figured it'd be a good change

i got banned for a seperate pr

File: code/modules/surgery/organs/tongue.dm (hunk 1 above)
Comment (BootlegBow) [reply code/modules/surgery/organs/tongue.dm:50]: i would do that if i had enough brain cells to code it but unfortunately i'm a fucking idiot

**Pull request closed without merging**